File: app/src/main/java/com/esorokin/lantector/utils/BuildUtils.java
package com.esorokin.lantector.utils;

import com.esorokin.lantector.BuildConfig;

public final class BuildUtils {
	private BuildUtils() {
		//utils
	}

	public static boolean isRelease() {
		return !BuildConfig.DEBUG;
	}

	public static boolean isDebug() {
		return BuildConfig.DEBUG;
	}

	public static boolean isTurnLogs() {
		return BuildConfig.IS_TURN_LOGS;
	}
}

File: app/src/main/java/com/esorokin/lantector/utils/BuildUtils.kt
package com.esorokin.lantector.utils

import com.esorokin.lantector.BuildConfig

object BuildUtils {
    val isRelease: Boolean
        get() = !BuildConfig.DEBUG

    val isDebug: Boolean
        get() = BuildConfig.DEBUG

    val isTurnLogs: Boolean
        get() = BuildConfig.IS_TURN_LOGS
}
